float32 comm_mode
float32 diff_mode
int16 update
---
float32 comm_mode
float32 diff_mode 